packages/camera/camera_android_camerax/lib/src/camera_state.dart

```diff
@@ -0,0 +1,121 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/services.dart';
+import 'package:meta/meta.dart';
+
+import 'camera_state_error.dart';
+import 'camerax_library.g.dart';
+import 'instance_manager.dart';
+import 'java_object.dart';
+
+/// The state of a camera.
+///
+/// See https://developer.android.com/reference/androidx/camera/core/CameraState.
+class CameraState extends JavaObject {
+  /// Constructs a [CameraState] that is not automatically attached to a native object.
+  CameraState.detached(
+      {super.binaryMessenger,
+      super.instanceManager,
+      required this.type,
+      this.error})
+      : super.detached();
+
+  /// The type of state that the camera is in.
+  final CameraStateType type;
+
+  /// The error that the camera has encountered, if any.
+  final CameraStateError? error;
+
+  /// Error code indicating that the camera device is already in use.
+  ///
+  /// See https://developer.android.com/reference/androidx/camera/core/CameraState#ERROR_CAMERA_IN_USE()
+  static int errorCameraInUse = 1;
```

Comment:
Blocking: I think all these error codes should be all caps and static final. 

Specifically making them final is blocking.